pom.xml

```diff
@@ -94,6 +94,11 @@
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>jakarta.activation</groupId>
+        <artifactId>jakarta.activation-api</artifactId>
+        <version>1.2.2</version>
+      </dependency>
```

Comment:
jenkins ships with a lesser version.  if this is bundled in the plugin it is never used as the plugin is not using a plugin first classloader, so the fix should be to exclude the version instead.
if it is not bundled - then this means the test classpaths are more likely to more even invalid that normal.